Src/PerformanceCollector/Perf.Shared.NetStandard/Implementation/QuickPulse/QuickPulseServiceClient.cs

```diff
@@ -261,6 +278,21 @@ private void WriteSamples(IEnumerable<QuickPulseDataSample> samples, string inst
                 ProcessCpuData[] topCpuProcesses =
                     sample.TopCpuData.Select(p => new ProcessCpuData() { ProcessName = p.Item1, CpuPercentage = p.Item2 }).ToArray();
 
+
+                bool perfCollectionSupported = false;
```

Comment:
Should we extract this code into its own method?

Reply:
Yes done.